nethserver-mail.spec

```
@@ -237,7 +239,9 @@ EOF
 
 cat >>imapsync.lst <<'EOF'
 %dir %{_nseventsdir}/%{name}-imapsync-update
-%dir %attr(0700,root,root) /var/log/imapsync
+%dir %attr(0700,vmail,vmail) /var/log/imapsync
+%dir %attr(0700,vmail,vmail) /var/lib/nethserver/imapsync
+%dir %attr(0700,vmail,vmail) /var/lib/nethserver/imapsync/secrets
```

The setgid bit causes any file created under the directory to be owned by the same group of the directory itself. This mechanism avoids the chown() calls and we are sure only members of vmail can access the file contents.

```suggestion
%dir %attr(1750,root,vmail) /var/lib/nethserver/imapsync
%dir %attr(1750,root,vmail) /var/lib/nethserver/imapsync/secrets
```

---

really a good idea...thank for the hint